outlines/models/transformers.py

```diff
@@ -105,6 +121,28 @@ def convert_token_to_string(self, token: str) -> str:
 
         return string
 
+    def format_prompt_into_conversation(self, prompt: str) -> List[Dict[str, str]]:
+        """
+        Template for one-turn chat.
+        """
+        return (
+            [self.SYSTEM_MESSAGE.copy(), self.USER_MESSAGE_TEMPLATE(prompt)]
+            if self.SYSTEM_MESSAGE is not None
+            else [self.USER_MESSAGE_TEMPLATE(prompt)]
+        )
```

Comment:
Not all models have a system message (e.g. phi-3)

Is there a way built into `transformers` to handle this? (I'm not aware of one)